internal/dag/builder_test.go

```diff
@@ -5160,7 +5262,7 @@ func TestDAGInsert(t *testing.T) {
 					VirtualHosts: virtualhosts(
 						securevirtualhost("b.example.com", sec1,
 							prefixroute("/", service(s1)),
-							prefixroute("/kuarder", service(s2)),
+							exactroute("/kuarder", service(s2)),
```

Comment:
FYI: Made these sorts of changes so we wouldn't have to assert on regex rules all over the place